cmd/src/campaigns_common.go

```diff
@@ -169,10 +169,6 @@ func campaignsExecute(ctx context.Context, out *output.Output, svc *campaigns.Se
 		defer specFile.Close()
```

Comment:
maybe in l.82 a doc change would be nice. Like it defaults to the authenticated user's namespace or so

Reply:
This is the ticket for the doc changes: https://github.com/sourcegraph/sourcegraph/pull/15664

Did you mean something else?

Reply:
This little docstring I meant https://github.com/sourcegraph/src-cli/blob/ec0a7ab72e2bc3d258b744260d8a3871057766ee/cmd/src/campaigns_common.go#L82

Reply:
Ah, good call. I added a note about the default.